docs/standard/components.md

```diff
@@ -1,80 +1,82 @@
-
-Here's a graphical overview of each of the previously mentioned components of .NET and how they fit.
-
-![All .NET Architectural Components Together](media/components.png)
-
-What follows is a brief explanation of each of the key components shown above.  
+A .NET app is developed for and runs in one or more *implementations of .NET*.  Implementations of .NET include the .NET Framework, .NET Core, and Mono for Xamarin. There is an API specification common to all implementations of .NET that is called the .NET Standard. This article gives a brief introduction to each of these concepts.
 
 ## .NET Standard
 
-The .NET Standard is a set of APIs which are implemented by a .NET runtime.
+The .NET Standard is a set of APIs that are implemented by the Base Class Library of a .NET implementation. More formally, it's a specification of .NET APIs that make up a uniform set of contracts that you compile your code against. These contracts are implemented in each .NET implementation. This enables portability across different .NET implementations, making it so that your code can effectively run everywhere.
 
-More formally, it is a specification of .NET APIs which make up a uniform set of contracts that you compile your code against.  These contracts have underlying implementations for each .NET runtime.  This enables portability across different .NET runtimes, making it so that your code can effectively "run everywhere".
-
-The .NET Standard is also a build target. You can currently target .NET Standard 1.0-2.0. If your code targets a version of the .NET Standard, it is guaranteed to run on any .NET runtime which implements that version.
+The .NET Standard is also a target framework. If your code targets a version of the .NET Standard, it can run on any .NET implementation which supports that version of .NET Standard.
 
 To learn more about the .NET Standard and how to target it, see the [.NET Standard](net-standard.md) topic.
 
-## .NET runtimes
+## .NET implementations
+
+Each implementation of .NET includes the following components:
 
-There are 3 primary .NET runtimes which Microsoft actively develops and maintains: .NET Core, .NET Framework, and Mono for Xamarin.
+- One or more runtimes. Examples: CLR for .NET Framework, CoreCLR and CoreRT for .NET Core.
+- A class library that implements the .NET Standard and may implement additional APIs. Examples: .NET Framework Base Class Library, .NET Core Base Class Library.
+- Optionally, one or more application frameworks. Examples: [ASP.NET](https://www.asp.net/), [Windows Forms](../framework/winforms/windows-forms-overview.md), and [Windows Presentation Foundation (WPF)](../framework/wpf/index.md) are included in the .NET Framework.
+- Optionally, development tools. Some development tools are shared among multiple implementations.
+
+There are three primary .NET implementations that Microsoft actively develops and maintains: .NET Core, .NET Framework, and Mono for Xamarin.
```

Comment:
I think the relationship of Mono & Xamarin is similar to .NET Framework & .NET Core in that Xamarin is a subset and has specific runtimes. /cc @mhutch

I assume we're trying to make the point here that we don't actively maintain Mono but only Mono for Xamarin?